src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
> The divider in fact can never get hold on the contained items

Why would you need that?

> Furthermore, a divider cannot possibly have a renderer, that e.g. wraps a div around the items to add an accordion-like expander or similar things.

Which is a good thing, right? The divider might not have any renderer at all. The list renderer has knowledge about the sequence of dividers, headings and items and can act accordingly. If wrapping divs depend on the list type, your group renderer would not be able to do this, too (or we end up with lots of group renderers). I think the list renderer should be responsible for wrapping divs.

To enable submenues in Dropdowns all I need is to allow Dropdowns in the item array. The Dropdown renderer would be able to happily render submenues.

> Furthermore, a divider cannot possibly have a renderer, that e.g. wraps a div around the items to add an accordion-like expander or similar things.

See above. The list renderer can do this.

> Speaking on a very high level, a group is strictly more powerful than a divider. This is because a divider does know nothing about its location, while a group knows the items it spans.

Similar like the Divider has no knowledge about the items, the group does have no knowledge about the list.